dom.bs

```diff
@@ -1253,6 +1253,9 @@ for discussion).
 
  <li><p>Set <var>event</var>'s {{Event/eventPhase}} attribute to {{Event/NONE}}.
 
+ <li><p>If {{Event/target}}'s <a for=tree>root</a> is a <a for=/>shadow root</a>, set
```

Comment:
then set*

Reply:
Thanks, done.

Will add a test case there for relatedTarget is in document tree while target is in shadow tree and event stops at shadow root to make sure both are set to null.